app/src/handlers/census/MetagameEventEventHandler.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ *  ### CENSUS RESPONSE ATTRIBUTES ####
+ "event_name":"MetagameEvent",
+ "experience_bonus":"",
+ "faction_nc":"",
+ "faction_tr":"",
+ "faction_vs":"",
+ "metagame_event_id":"",
+ "metagame_event_state":"",
+ "timestamp":"",
+ "world_id":"",
+ "zone_id":""
+ * ### END ###
+ **/
+
+import {injectable} from 'inversify';
+import EventHandlerInterface from '../../interfaces/EventHandlerInterface';
+import {GenericEvent} from 'ps2census/dist/client/utils/PS2Events';
+import {getLogger} from '../../logger';
+import config from '../../config';
+import {jsonLogOutput} from '../../utils/json';
+import MetagameEventEvent, {MetagameEventState} from './events/MetagameEventEvent';
+
+declare type Alert = {
+    worldId: number;
+    instanceId: number;
+};
+
+@injectable()
+export default class MetagameEventEventHandler implements EventHandlerInterface {
```

Comment:
`MetagameEventEventHandler`, heh that's unfortunate naming on our part, would be what I would have used though!